nesta/core/batchables/nlp/bert_vectorize/run.py

```diff
@@ -65,15 +65,29 @@ def run():
     # Convert text to vectors
     embeddings = model.encode(docs)
     # Write to output
-    out_data = [{"article_id": _id, "vector": embedding.tolist()}
+    out_data = [{id_field: _id, "vector": ["%.5f" % v for v in embedding.tolist()]}
                 for _id, embedding in zip(ids, embeddings)]
     Base = get_base_from_orm_name(out_module)
     out_class = get_class_by_tablename(out_module, out_tablename)
     insert_data("BATCHPAR_config", "mysqldb", db_name, Base,
-                out_class, out_data, low_memory=True)
+                out_class, out_data, low_memory=True,
+                insert_chunksize=10)
```

Comment:
`insert_chunksize` is a new feature, so "upgrading" the insertion which minimises bottlenecks with MySQL due to very large chunks (I've found that 10, 100, 1000 all have the same run time, so settled on the smallest)